molgenis-data-rest/src/main/java/org/molgenis/data/rest/v2/RestControllerV2.java

```diff
@@ -91,17 +99,32 @@ static UnknownEntityException createUnknownEntityException(String entityName)
 		return new UnknownEntityException("Operation failed. Unknown entity: '" + entityName + "'");
 	}
 
+	static MolgenisDataAccessException createNoReadPermissionOnEntityException(String entityName)
+	{
+		return new MolgenisDataAccessException("No read permission on entity " + entityName);
+	}
+
+	static MolgenisDataException createNoWriteCapabilitiesOnEntityException(String entityName)
+	{
+		return new MolgenisRepositoryCapabilitiesException("No write capabilities for entity " + entityName);
+	}
+
+	static DuplicateEntityException createDuplicateEntityException(String entityName)
```

Comment:
I mean to refactor them using "inline method" refactoring. That is, to call the constructor directly where you now call the method that calls the constructor for you.